conf/fsb-types.yaml

```diff
@@ -1,12 +1,39 @@
 - key: SAC
+  schemeId: GovernmentStatisticalService
```

Comment:
This will potentially lead to unsync. We have to manage schemes in this list and then in schemes yams file. 

I think it should go as-is, but in my story, I'll remove this file and use extended schemes structure:
https://github.com/hmrc/fset-faststream/commit/5a50eb034b3a789761094d57e49257d41e0dd718#diff-1e6cb5c6d8ee215686f6a13e5a2b99b6
If there are any objections, please let me know.